charts/linkerd2/templates/_nodeselector.tpl

```diff
@@ -0,0 +1,8 @@
+{{- define "linkerd.node-selector" }}
+nodeSelector:
+{{- if .NodeSelector }}
+{{- toYaml .NodeSelector | trim | nindent 2 }}
+{{- else }}
+  beta.kubernetes.io/os: linux
```

Comment:
WDYT about just setting the value to this and getting rid of the if/else?

Reply:
I agree :+1: